scripts/generate-changelog.js

```diff
@@ -1,21 +1,28 @@
 'use strict'
 
-const exec = require('child_process').exec
+const { exec } = require('child_process')
 
 const fetchTags = () =>
```

Comment:
I think we could add some comments about how it works/what's happening here, it's a bit difficult to understand, for example explaining the format used for git log and how it's used in the regexp.